pkg/autoscaler/k8sclient/k8sclient.go

```diff
@@ -209,6 +214,19 @@ func (k *k8sClient) GetClusterStatus() (clusterStatus *ClusterStatus, err error)
 	return clusterStatus, nil
 }
 
+// filter inappropriate nodes
+func filterSkippedNode(node *apiv1.Node) (skipped bool) {
+	labels := node.GetLabels()
+	for label, value := range labels {
+		// virtual-kubelet node has large amount of capacity which will mislead cluster-proportional-autoscaler
+		if label == "type" && value == "virtual-kubelet" {
```

Comment:
unprefixed labels (e.g. "type" vs "virtual-kubelet.io/type") are strictly reserved for end users and MUST NOT be used by infrastructure.